src/NHibernate.Test/TestCase.cs

```diff
@@ -447,6 +447,19 @@ protected DateTime RoundForDialect(DateTime value)
 				{"locate", new HashSet<System.Type> {typeof (SQLiteDialect)}},
 				{"bit_length", new HashSet<System.Type> {typeof (SQLiteDialect)}},
 				{"extract", new HashSet<System.Type> {typeof (SQLiteDialect)}},
+				{
+					"bxor",
+					new HashSet<System.Type>
+					{
+						// Could be supported like Oracle, with a template
+						typeof (SQLiteDialect),
+						// Could be supported by overriding registration with # instead of ^
+						typeof (PostgreSQLDialect),
+						typeof (PostgreSQL81Dialect),
+						typeof (PostgreSQL82Dialect),
+						typeof (PostgreSQL83Dialect)
```

Comment:
So, the 5 lines above are obsolete, in regard to current master state. Should it be change as part of the merge? Or as a later commit? (If change as a later commit, as already written somewhere else, I intend to do it while adding support for bxor in `SQLiteDialect`.)